src/test/java/seedu/address/model/recipe/CompletionTimeTest.java

```diff
@@ -34,14 +34,19 @@ public void toString_test() {
     }
 
     @Test
-    public void equalityCheck() {
+    public void equals_test() {
```

Comment:
Just a reminder, if we are standardizing our test method names to be more specific/detailed. Then we should change the equals_test method name.

```suggestion
    public void test_equalityCheckForValidAndInvalidCompletionTime() {
```